Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:12.17.0-alpine
```

Comment:
Similar question on this PR when adding a Dockerfile. There should be a `README.md` update as well if we are going to include this change in a PR to the forked repo.

Reply:
@hankwallace 
I will update README for forked repo and will add a command for building and running docker.

Reply:
@hankwallace Updated this in REAMD file.